docs/import/manage_disaster.js

```diff
@@ -515,12 +517,21 @@ function initializeScoreSelectors(states, stateAssets) {
   }
 
   // For each asset type, add select for all assets for each state.
-  for (const {idStem, propertyPath, expectedColumns} of scoreAssetTypes) {
+  for (const {idStem,
+              propertyPath,
```

Comment:
also add hasGeometry to this sig? (and others?)

Reply:
Think I got all of them!